xbmc/cores/RetroPlayer/guibridge/GUIGameRenderManager.cpp

```diff
@@ -100,6 +116,11 @@ std::shared_ptr<CGUIGameVideoHandle> CGUIGameRenderManager::RegisterDialog(GAME:
   return std::make_shared<CGUIGameVideoHandle>(*this);
 }
 
+std::shared_ptr<CGUIGameSettingsHandle> CGUIGameRenderManager::RegisterDialog(GAME::CDialogGameAdvancedSettings &dialog)
+{
+  return std::make_shared<CGUIGameSettingsHandle>(*this);
+}
+
```

Comment:
Where is the dialog param used or why do you need it?

Reply:
A parameter is needed for the game window and game controls, and I just copied the pattern for game settings. It's not needed so I'll remove it.